packages/core-data/src/entity-provider.js

```diff
@@ -200,18 +245,24 @@ export function useEntityBlockEditor( kind, name, { id: _id } = {} ) {
 			edits.content = ( { blocks: blocksForSerialization = [] } ) =>
 				__unstableSerializeAndClean( blocksForSerialization );
 
-			editEntityRecord( kind, name, id, edits );
+			registry.batch( () => {
+				updateFootnotes( edits.blocks );
```

Comment:
Do we always need to update the footnotes for all block editor providers? Is there a way to only do that when a "post type" supports footnotes, should we introduce a new flag to CPT registration to make footnotes support optional? opt-in ideally or opt-out.

Reply:
Added a check for meta.footnotes === undefined, which means that footnotes meta is not registered. That can act as our opt-in support flag.